### PR TITLE
xqilla/2.3.4 Pass build flag to configure for armv8 native build.

### PR DESCRIPTION
**Pull request by bowb**
Specify library name and version:  **xqilla/2.3.4**

 Included autotools is a bit out of date.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
Too fragile.
Take a look at `odbc` recipe for example to see how to update `config.guess` & `config.sub`:
https://github.com/conan-io/conan-center-index/blob/68ced7ba4535391596812cb8d85e1f32b3f71d57/recipes/odbc/all/conanfile.py#L49-L50
https://github.com/conan-io/conan-center-index/blob/68ced7ba4535391596812cb8d85e1f32b3f71d57/recipes/odbc/all/conanfile.py#L75-L77
https://github.com/conan-io/conan-center-index/blob/68ced7ba4535391596812cb8d85e1f32b3f71d57/recipes/odbc/all/conanfile.py#L82-L85

Adapt those lines to copy `config.guess` & `config.sub` in the proper folder(s) of xqilla.

**Review by SpaceIm — APPROVED** [review]

**Review by SSE4 — DISMISSED** [review]
the title is a bit confusing, as it doesn't add any new configure flags, but changes are okay

File: recipes/xqilla/all/conanfile.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from conans import AutoToolsBuildEnvironment, ConanFile, tools
 from conans.errors import ConanInvalidConfiguration
 import os
+import shutil
 
 required_conan_version = ">=1.40.0"
```
Comment (SpaceIm) [inline recipes/xqilla/all/conanfile.py:6]: min version is too high, 1.33.0 is sufficient

**Comment by SpaceIm** [comment]
By the way, it would be nice to at least support all non-Windows platform in this recipe. It builds just fine on Macos with few fixes in their fragile build files, there is no reason to support only Linux.

**Review by madebr — APPROVED** [review]

**Comment by bowb** [comment]
> 



> By the way, it would be nice to at least support all non-Windows platform in this recipe. It builds just fine on Macos with few fixes in their fragile build files, there is no reason to support only Linux.



> By the way, it would be nice to at least support all non-Windows platform in this recipe. It builds just fine on Macos with few fixes in their fragile build files, there is no reason to support only Linux.

My local Macos build fails during linkage. Did you change something else to get Macos to build?

**Comment by SpaceIm** [comment]
> My local Macos build fails during linkage. Did you change something else to get Macos to build?

Yes, you have to propagate frameworks dependencies of xerces-c, because they don't rely on something robust like pkg-config. You can fix that in another PR.

**Review by SpaceIm — APPROVED** [review]

**Review by madebr — APPROVED** [review]

**Pull request merged**